packages/engine/Specs/Core/defaultValueSpec.js

```diff
@@ -1,15 +1,15 @@
-import { defaultValue } from "../../index.js";
-
-describe("Core/defaultValue", function () {
-  it("Works with first parameter undefined", function () {
-    expect(defaultValue(undefined, 5)).toEqual(5);
-  });
-
-  it("Works with first parameter null", function () {
-    expect(defaultValue(null, 5)).toEqual(5);
-  });
-
-  it("Works with first parameter not undefined and not null", function () {
-    expect(defaultValue(1, 5)).toEqual(1);
-  });
-});
+// import { defaultValue } from "../../index.js";
```

Comment:
This can probably be deleted completely right?

Reply:
Yeah This can be deleted.